Kafka/src/main/java/io/deephaven/kafka/ingest/KafkaIngester.java

```diff
@@ -320,7 +320,7 @@ private boolean pollOnce(final Duration timeout) {
         } catch (Exception ex) {
             log.error().append(logPrefix).append("Exception while polling for Kafka messages:").append(ex)
                     .append(", aborting.");
-            return false;
+            return true;
```

Comment:
Error message no longer matches the return. Also, there's no feedback here. What do we expect to be the result?

Reply:
My confusion.